ddmrp/views/stock_buffer_view.xml

```diff
@@ -89,6 +89,15 @@
                     type="object"
                     attrs="{'invisible':[('rfq_outside_dlt_qty', '=', 0)]}"
                 />
+                <field name="rfq_inside_dlt_qty" invisible="1" />
+                <button
+                    title="Some RFQ quantities are inside of the DLT Horizon and may require rescheduling.
```

Comment:
Change to:

"Some RFQs are inside of the DLT Horizon and waiting validation to be accounted into the NFP. Press this button to display them."